### ISSUE TITLE
Link error: duplicate symbol _OBJC_IVAR_$_AppDelegate._window 

### ISSUE THREAD

**Issue by mironal** [devedup/FlickrKit#36]
I was install FlickrKit (v1.0.6) using cocoapods. 
After that, I got link error. :arrow_down: 

```
duplicate symbol _OBJC_IVAR_$_AppDelegate._window in:
    /Users/miro/Library/Developer/Xcode/DerivedData/feather-baxuldxkouoqtreylprgzubzrwmc/Build/Intermediates/feather.build/Debug-iphonesimulator/feather.build/Objects-normal/i386/AppDelegate.o
    /Users/miro/Library/Developer/Xcode/DerivedData/feather-baxuldxkouoqtreylprgzubzrwmc/Build/Products/Debug-iphonesimulator/libFlickrKit.a(AppDelegate.o)
duplicate symbol _OBJC_CLASS_$_AppDelegate in:
    /Users/miro/Library/Developer/Xcode/DerivedData/feather-baxuldxkouoqtreylprgzubzrwmc/Build/Intermediates/feather.build/Debug-iphonesimulator/feather.build/Objects-normal/i386/AppDelegate.o
    /Users/miro/Library/Developer/Xcode/DerivedData/feather-baxuldxkouoqtreylprgzubzrwmc/Build/Products/Debug-iphonesimulator/libFlickrKit.a(AppDelegate.o)
duplicate symbol _OBJC_METACLASS_$_AppDelegate in:
    /Users/miro/Library/Developer/Xcode/DerivedData/feather-baxuldxkouoqtreylprgzubzrwmc/Build/Intermediates/feather.build/Debug-iphonesimulator/feather.build/Objects-normal/i386/AppDelegate.o
    /Users/miro/Library/Developer/Xcode/DerivedData/feather-baxuldxkouoqtreylprgzubzrwmc/Build/Products/Debug-iphonesimulator/libFlickrKit.a(AppDelegate.o)
ld: 3 duplicate symbols for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Probably.. I think because the library contains `AppDelegate.h`, `AppDelegate.m` and `main.m` files.

![image](https://cloud.githubusercontent.com/assets/537587/11027607/669466ae-86fa-11e5-827b-21dd81e3eff0.png)

**Comment by topa51** [comment]
I have same issue,
I've changed my appDelegate class, to Prefix_Appdelegate class. This solve an issue.
But AppDelegate should't be included to pods.

**Comment by devedup** [comment]
Sorry. I'll take a look and do another release.

**Comment by devedup** [comment]
Fixed and released 1.0.7. It's because I had moved the ModelGenerator classes inside the main Classes directory - they didn't need to be in there, so I'm moved them out again so that they aren't packaged with FlickrKit.

**Comment by mironal** [comment]
thanks!! :+1:
